backend/infrahub/core/attribute.py

```diff
@@ -599,6 +600,26 @@ class Boolean(BaseAttribute):
     type = bool
 
 
+# TODO: Add properties for different datetime format outputs to mimic IP property types for GraphQL
```

Comment:
is this something we could create an issue for instead of leaving it in here as a todo?
I'd make the issue for you, but I don't know exactly what the requirements are here

Reply:
I opened the following issue and removed the `TODO` comment.

https://github.com/opsmill/infrahub/issues/3626